operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java

```diff
@@ -364,6 +366,24 @@ protected Step patchPod(V1Pod currentPod, Step next) {
             new V1Patch(patchBuilder.build().toString()), patchResponse(next));
   }
 
+  private Step updateCurrentPod(V1Pod currentPod, Step next) {
+    return createProgressingStep(updatePod(currentPod, next));
+  }
+
+  protected Step updatePod(V1Pod currentPod, Step next) {
+    JsonPatchBuilder patchBuilder = Json.createPatchBuilder();
+    String baseName = "/metadata/labels/";
+    if (!getPodLabels().containsKey(INTROSPECTION_STATE_LABEL)) {
+      patchBuilder.add(baseName + INTROSPECTION_STATE_LABEL, getDomain().getSpec().getIntrospectVersion());
+    } else {
+      patchBuilder.replace(baseName + INTROSPECTION_STATE_LABEL, getDomain().getSpec().getIntrospectVersion());
```

Comment:
Minor minor nit: combine repeated patch string concatenation code into a single local variable...

Reply:
Or better, into a method call. But I agree that this code is needless duplication.